.ci/windows.groovy

```diff
@@ -66,7 +66,7 @@ pipeline {
         gitCheckout(basedir: "${BASE_DIR}", githubNotifyFirstTimeContributor: true)
         stashV2(name: 'source', bucket: "${JOB_GCS_BUCKET}", credentialsId: "${JOB_GCS_CREDENTIALS}")
         dir("${BASE_DIR}"){
-          loadConfigEnvVars()
+          //loadConfigEnvVars()
```

Comment:
Did you mean to leave this comment in?

Reply:
Yes, I didn't want to remove anything in case we want to merge this pipeline into the `Jenkinsfile`